packages/eslint-plugin-circuit-ui/no-renamed-props/index.ts

```diff
@@ -301,22 +301,16 @@ const configs: (Config & { components: string[] })[] = [
         const current = getAttributeValue(attribute);
 
         if (current === 'highlight') {
-          const replacement = `as="strong" weight="bold"`;
-          const weightAttribute = findAttribute(node, 'weight');
           const asAttribute = findAttribute(node, 'as');
+          const weightAttribute = findAttribute(node, 'weight');
+          const replacement = asAttribute ? 'weight="semibold"' : `as="strong"`;
           context.report({
             node: attribute,
             messageId: 'bodyVariant',
             data: { component, current, replacement },
             fix: weightAttribute
```

Comment:
does this mean that if we have a weight attribute besides the variant, we do nothing ?

Reply:
We don't apply a fix, but the issue is still flagged. I decided to add this back in in case of custom props on styled components.